hieradata/vagrant_credentials.yaml

```diff
@@ -26,6 +26,9 @@ collectd::plugin::postgresql::password: 'password'
 govuk::apps::bouncer::postgresql_role::password: 'sKnvsmsAEeviNNl3Ded90NzxoT7eFI6Qd1cOVmaA'
 govuk::apps::ckan::db::password: 'goh8Deev3peki7liuxool7mo'
 govuk::apps::ckan::db_password: "%{hiera('govuk::apps::ckan::db::password')}"
+govuk::apps::ckan::google_analytics_access_token: "%{hiera('govuk::apps::ckan::db::google_analytics_access_token')}"
```

Comment:
Have the secrets definitely been added to the `govuk::apps::ckan::db` (database) namespace?